src/structure/viewer.ts

```diff
@@ -390,19 +396,16 @@ export class MoleculeViewer {
                 cstyle: { hidden: true },
             });
         }
-        assignBonds(this._current.model.selectedAtoms({}) as $3Dmol.AtomSpec[]);
-        this._current.model.addAtomSpecs(['index']);
+        assignBonds(this._current.model.selectedAtoms({}));
```

Comment:
Can we drop the explicit type casting in the new version?

Reply:
yes, the new type declarations are more accurate, so we don't have to do this manually anymore